configserver/src/main/java/com/yahoo/vespa/config/server/session/PrepareParams.java

```diff
@@ -273,6 +306,7 @@ public static PrepareParams fromHttpRequest(HttpRequest request, TenantName tena
                             .verbose(request.getBooleanProperty(VERBOSE_PARAM_NAME))
                             .timeoutBudget(SessionHandler.getTimeoutBudget(request, barrierTimeout))
                             .applicationId(createApplicationId(request, tenant))
+                            .tags(Tags.fromString(request.getProperty(TAGS_PARAM_NAME)))
```

Comment:
This PR doesn't set this value.

Reply:
How do you figure that? Afaik this payload is autogenerated from com.yahoo.vespa.hosted.controller.api.application.v4.model.DeploymentData, which sets tags on line 655 in ApplicationController.

Reply:
The `DeploymentData` has the tags, but they're not read anywhere. The implementation of the `ConfigServer` needs to do that.

Reply:
Sorry, this wasn't the relevant one. The `fromJson` is it. I'll make a PR for you :)

Reply:
The fromJson does reads the tags already?

Reply:
Yes, but they're not written anywhere.